main_ovs.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import nltk
+import re
+import string
+import json
+
+
+class FileReader:
+    @staticmethod
+    def read_file(name):
+        """ func to read from file """
+        with open(name) as fl:
+            return fl.readlines()
+
+
+class JsonWriter:
+    @staticmethod
+    def write_json(name, jsn):
+        with open(name, 'w') as file:                   # serialization
+            json.dump(jsn, file)
+
+
+class RemoverUnwantedWords:
+    @staticmethod
+    def remove_words(expr, line):
+        """ func for cleaning by regex """
+        return re.sub(expr, '', line)
+
+
+class CheckerForMeaning:
+    @staticmethod
+    def find_trash(clean_line, inf_dict):
+        """ find contextless word """
+        tokens = clean_line.split(' ')
+
+        clean_tokens = []
+        for i in range(len(tokens)):                    # removing punctuation and " "
+            if len(tokens[i]) < 2:
+                continue
+            while True:
+                if tokens[i][-1] in string.punctuation:
+                    tokens[i] = tokens[i][:-1]
+                else:
+                    clean_tokens.append(tokens[i])
+                    break
+
+        clean_tokens = set(clean_tokens)
+        for i in clean_tokens:                          # check for meaning
+            if nltk.wsd.lesk(tokens, i) is None:
+                inf_dict['orphan_tokens'].append(i)
+        return inf_dict
```

Comment:
If you **absolutely have to** mutate the data sent you as an argument, don't return this very data from your method - your method should either have an "out" parameter to mutate, or a return value, not both. Having both, you confuse a reader of your method.